lib/styles/settings/typography/_tokens.scss

```diff
@@ -0,0 +1,46 @@
+$typography-font-family: var(--typography-font-family);
+
+$typography-font-size-base: var(--typography-font-size-base);
+$typography-font-size-s: var(--typography-font-size-s);
+$typography-font-size-xs: var(--typography-font-size-xs);
+$typography-font-size-2xs: var(--typography-font-size-2xs);
+
+$typography-font-weight-bold: var(--typography-font-weight-bold);
+
+$typography-line-height-2xs: var(--typography-line-height-2xs);
+$typography-line-height-3xs: var(--typography-line-height-3xs);
+$typography-line-height-4xs: var(--typography-line-height-4xs);
+
+$typography-letter-spacing-s: var(--typography-letter-spacing-s);
+
+body {
+  font-family: $typography-font-family;
+  font-size: $typography-font-size-base;
+}
+
+@mixin labelLDefaultBold() {
+  font-family: $typography-font-family;
+  font-weight: $typography-font-weight-bold;
+  line-height: $typography-line-height-2xs;
+  font-size: $typography-font-size-s;
+  letter-spacing: $typography-letter-spacing-s;
+  --token: label-l-default-bold;
```

Comment:
I'm not sure if it was your intention but custom attributes are inherited by child nodes.
We need to think about risks of this feature

Reply:
@kvas-damian there is one fix, but it increased code size:
```
--token: label-l-default-bold;
--token: initial;
```
Parent will have visible token (but with line-through) and childrens will have `--token: initial`.

Reply:
I think we should inform about that fact UX.

Reply:
Maybe this inheritance is not so stupid. Example:
1. Parent has `labelMDefaultBold();` with `--token: label-m-default-bold;` 
2. Children has no styles
3. Children of children inherits all parent styles and overwrites only font-size. So we will see in devtools:
a. Overwritten font size
b. all `labelMDefaultBold` rules with info where they come from - `--token: label-m-default-bold;`

Reply:
is 3. an advantage or disadvantage? What UX would like to see?

Reply:
> Parent will have visible token (but with line-through) and childrens will have `--token: initial`.

I'm not sure if it's intuitive solution